.gitignore

```diff
@@ -28,5 +28,3 @@ node_modules
 
 .DS_Store
 *.iml
-
-dist
```

Comment:
@beenanner it is a quick fix to make sure we can ship the released version.

When we make the release tag and automatically creates the release bundle for Github, it takes the files that are in checked in, so we need a way to add the file to the release (upload it ourselves)? Ready made solutions are welcomed.